code/modules/jobs/job_types/scientist.dm

```diff
@@ -21,6 +21,12 @@
 
 	family_heirlooms = list(/obj/item/toy/plush/slimeplushie)
 
+	mail_goodies = list(
+		/obj/item/raw_anomaly_core/random = 10,
+		/obj/item/disk/tech_disk/spaceloot = 2,
+		/obj/item/camera_bug = 1
```

Comment:
I forget, can you still deconstruct camera bugs to unlock illegal tech?

that might be fine, of course- just lettin' ya know